src/ui/startup_dialog.py

```
@@ -26,6 +26,8 @@
 )
 
 from core.experiment_manager import ExperimentManager, Experiment
+from ui.settings_dialog import SettingsDialog
+from ui.settings_dialog import SettingsDialog
```

Ruff (0.14.14)

[error] Redefinition of unused `SettingsDialog` from line 29: `SettingsDialog` redefined here

Remove definition: `SettingsDialog`

(F811)